apiserver/Makefile

```diff
@@ -42,10 +42,12 @@ APISERVER_BUILD_DATE?=$(shell date -u +'%FT%T%z')
 APISERVER_GIT_REVISION?=$(shell git rev-parse --short HEAD)
 APISERVER_GIT_DESCRIPTION?=$(shell git describe --tags)
 
-LDFLAGS = -X $(PACKAGE_NAME)/cmd/apiserver/server.VERSION=$(APISERVER_VERSION) \
+LDFLAGS =-ldflags "\
```

Comment:
@rene-dekker You moved `-ldflags` out of `LDFLAGS` into the `go build` command line in a previous PR (for example, this [one](https://github.com/projectcalico/calico/pull/7383/files#diff-72be7afda7c33dd9a23c823b2824820d758e68e8f467465dc58f26b52a942471R35)). This change moved them back. Any reasons why we are moving it back and forth?

Reply:
@hjiawei Thank you for pointing this out. I have removed `-ldflags` again from `LDFLAGS`.